yorm/test/test_utilities.py

```diff
@@ -114,6 +114,26 @@ def test_exception_when_file_not_expected_but_found(self):
         with pytest.raises(exceptions.FileAlreadyExistsError):
             utilities.sync(self.Sample(), "sample.yml", existing=False)
 
+    def test_file_exts(self):
+        # Standards (argument, expection)
+        for ext in [('yml', 'yml'), ('yaml', 'yml'), ('json', 'json')]:
```

Comment:
This might work better as a parameterized test: https://pytest.org/latest/parametrize.html